doc/source/serve/performance.md

```diff
@@ -150,6 +150,10 @@ There are handful of ways to address these issues:
 (serve-performance-async-methods)=
 ### Using `async` methods
 
+:::{note}
+According to the [FastAPI documentation](https://fastapi.tiangolo.com/async/#very-technical-details), `def` endpoint function will be called in a separate threadpool, you might observe many requests running at the same time inside one replica, and this scenario might cause the OOM or resource starvation. In this case, you can try to use `async def` and tune `max_concurrent_queries` to control the workload performance.
```

Comment:
```suggestion
According to the [FastAPI documentation](https://fastapi.tiangolo.com/async/#very-technical-details), `def` endpoint functions will be called in a separate threadpool, so you might observe many requests running at the same time inside one replica, and this scenario might cause OOM or resource starvation. In this case, you can try to use `async def` and tune `max_concurrent_queries` to control the workload performance.
```
By "tune" do we just mean increase? (i.e. if the user sees OOM/resource starvation -> user should increase max_concurrent_queries). If so let's just say "increase".

Reply:
Directly remove `max_concurrent_queries` for now, it is mentioned in the following section clearly.

Reply:
Why are we removing `max_concurrent_queries`?

Reply:
it is mentioned the `max_concurrent_queries` in the following section. (redundant info)

Reply:
Not directly related to the `def` and `async def` information